apps/dashboard_reports/static/dashboard_reports/styles/style.css

```
@@ -0,0 +1,230 @@
+  font-size: 20px;
+  font-weight: 600;
+}
+
+.header-table td.description {
+  font-size: 10px;
+  min-height: 24px;
+}
+
+.data-table {
+  border-collapse: collapse;
+  page-break-inside: avoid;
```

Stylelint (17.12.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)